standup.md

```diff
@@ -0,0 +1,134 @@
+a##Standup Notes:
```

Comment:
Im galdd yall took standup notes... standups and retrospectives are a big part of working dev teams.. and if the conversations are not doucmented it can be easy to forget what / when / and why desicions were made